CMakeLists.txt

```diff
@@ -182,18 +185,18 @@ FetchContent_Populate(${extension_name}
  )
 list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
-#SlicerIMSTK
-set(extension_name "SlicerIMSTK")
-set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
-FetchContent_Populate(${extension_name}
- SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
- GIT_REPOSITORY https://github.com/KitwareMedical/SlicerIMSTK.git
- GIT_TAG        cd37cd73f853cf3cb09b9611d9cb715d5f84613c
- GIT_PROGRESS   1
- QUIET
- )
-message(STATUS "Remote - ${extension_name} [OK]")
-list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
+# #SlicerIMSTK
+# set(extension_name "SlicerIMSTK")
+# set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
+# FetchContent_Populate(${extension_name}
+#  SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
+#  GIT_REPOSITORY https://github.com/KitwareMedical/SlicerIMSTK.git
+#  GIT_TAG        cd37cd73f853cf3cb09b9611d9cb715d5f84613c
+#  GIT_PROGRESS   1
+#  QUIET
+#  )
+# message(STATUS "Remote - ${extension_name} [OK]")
+# list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
```

Comment:
```suggestion
```

Reply:
We hope to use IMSTK at some point in the future, and leaving this code in as comments is intended to give us a head start at that time.